v1/plugin/flags.go

```diff
@@ -0,0 +1,56 @@
+package plugin
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/urfave/cli"
+)
+
+var (
+	flConfig = cli.StringFlag{
+		Name:  "config",
+		Usage: "config file to use",
+	}
+	// If no port was provided we let the OS select a port for us.
+	// This is safe as address is returned in the Response and keep
+	// alive prevents unattended plugins.
+	flPort = cli.StringFlag{
+		Name:  "port",
+		Usage: "port to listen on",
+	}
+	flLogLevel = cli.IntFlag{
+		Name:  "log-level",
+		Usage: "log level - 0:panic 1:fatal 2:error 3:warn 4:info 5:debug",
+		Value: 2,
+	}
+	flPprof = cli.BoolFlag{
+		Name:  "pprof",
+		Usage: "set pprof",
+	}
+	flTLS = cli.BoolFlag{
+		Name:  "tls",
+		Usage: "enable TLS",
+	}
```

Comment:
Good catch.  246fd7c477f653376ed8a26a6a087efa2e967cbf fixes it.

![img](https://www.evernote.com/l/AA3HUxNDsHZHaL6xm1ScjTTMk1Nf-Mt5JAQB/image.png)